### PR TITLE
{math}[intel/2019b] gmsh v4.5.6 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/c44d50a93841268d90da2add7b448192 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/b15fd48fc838f0c4910b064b30be1ae9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3129.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e94ab9fe3b23abe86b5b26f34d239e8c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/bfabcdc110b3765a43ebb5e42b9b1f3d for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node154.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/e68ab9344e2cc921155ac7d39d1f9cec for a full test report.

File: easybuild/easyconfigs/g/gmsh/gmsh-4.5.6-intel-2019b-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.5.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://gmsh.infio'
```
Comment (lexming) [inline easybuild/easyconfigs/g/gmsh/gmsh-4.5.6-intel-2019b-Python-2.7.16.eb:7]: There is a typo in the URL
```suggestion
homepage = 'https://gmsh.info'
```

**Comment by jhein32** [comment]
Kenneth, since this is also python 2.7.x, why doen't this one need the `local_pymajmin` gymnastics we did for gmsh-3.0.6-foss-2017b-Python-2.7.14.eb in #10476

**Comment by boegel** [comment]
@jhein32 Newer version where they probably use a differrent way of finding the Python to use (3.0.6 uses the deprecated `FindPythonInterp` mechanism from CMake, see https://cmake.org/cmake/help/v3.13/module/FindPythonInterp.html)

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/53b8f8844589b75e3ac47c4cda5d775e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/242f9baeb006825518f0110f8ef3b743 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**